include/seqan3/alphabet/concept_pre.hpp

```diff
@@ -139,6 +143,10 @@ constexpr auto alphabet_size_v = alphabet_size<alphabet_type>::value;
  * \param alph The alphabet letter that you wish to assign to.
  * \param rank The rank you wish to assign.
  * \returns A reference to `alph` or a temporary if `alph` was a temporary.
+ *
+ * \details
+ * \attention This is a concept requirement, not an actual function (however types satisfying this concept
+ * will provide an implementation).
```

Comment:
well, hard to say; types that **satisfy** the concept **must/will** provide –– types that **aim/wish to satisfy** the concept **shall** provide 🤓